doc/lush.txt

```diff
@@ -361,42 +361,41 @@ extends({parsed_lush_spec, ...}).with(spec)                  *lush-extends-with*
 `extends({parsed_lush_spec, ...})` *must* be proceeded by `with(spec)`, and
 returns a parsed_lush_spec.
 
-If you have an existing spec that you wish to modify, you may wish to use
+If you have an existing spec that you wish to modify, you may want to use
 `extends.with`.
 
 Potential use cases:
 
-  - You like a lush theme you got from online, but want to change a few
-    specific parts of it, such as the comment style, or the background
-    color.
+  - You like a lush theme you got online, but want to change a few specific
+    parts of it, such as the comment style, or the background color.
 
   - You are writing your own theme and want to make a small tweaks to create a
-    variant, for example a high-contrast or colorbind safe mode.
+    variant, for example a high-contrast or colorblind safe mode.
 
 >
-  -- you may have installed this theme via your package manager or simliar.
-  -- well behaving lush themes should be importable via this manner.
-  -- or it might be your 'main' theme file if you are writing a variant.
+  -- You may have installed this theme via your package manager or similar
+  -- (good lush themes should be portable), or you might be writing a variant
+  -- for your main theme.
   local some_theme = require('lush_theme.some_theme')
 
-  -- maybe you also want to include a community extension that patches
-  -- some mising styles in the original theme.
+  -- Maybe you also want to include a community extension that patches
+  -- some missing styles in the original theme.
   local some_theme_lsp = require('lush_theme.some_theme_lsp')
 
   local spec = lush.extends({some_theme, some_theme_lsp}).with(function()
-    -- note it is also valid to return an empty spec, which is functionally
-    -- equivilent to calling merge({some_theme}).
+    -- It's also valid to return an empty spec. That's functionally
```

Comment:
> It's also valid to return an empty spec. That's functionally equivalent to calling merge({some_theme}).

I think I prefer the more declarative "It is also valid to".

Also "That's" isn't wrong but I thing its a bit awkward. Hard to express why, it's almost like that's is past tense. 

Probably

> "This is functionally equivalent calling merge({some_theme})."

 reads better *and* connects better to the `return {}`?

Reply:
Yes, that also works.